main.tf

```diff
@@ -149,7 +150,7 @@ resource "aws_cloudwatch_event_target" "security_reporter_rule_target" {
 }
 
 resource "aws_ssm_parameter" "security_controls_api_host" {
-  name  = local.ps_key_security_controls_api_host
+  name  = "${var.ps_root_path}${var.ps_key_security_controls_api_host}"
```

Comment:
@dsv9818 where are the variables defined, I don't see the variables.tf file in the PR